core-services/image-mirroring/openshift/mapping_origin_4_12

```diff
@@ -272,6 +272,7 @@ registry.ci.openshift.org/origin/4.12:recycler quay.io/openshift/origin-recycler
 registry.ci.openshift.org/origin/4.12:redhat-subscription-injection-webhook quay.io/openshift/origin-redhat-subscription-injection-webhook:4.12 quay.io/openshift/origin-redhat-subscription-injection-webhook:4.12.0 quay.io/openshift/origin-redhat-subscription-injection-webhook:latest
 registry.ci.openshift.org/origin/4.12:sdn quay.io/openshift/origin-sdn:4.12 quay.io/openshift/origin-sdn:4.12.0 quay.io/openshift/origin-sdn:latest
 registry.ci.openshift.org/origin/4.12:secondary-scheduler-operator quay.io/openshift/origin-secondary-scheduler-operator:4.12 quay.io/openshift/origin-secondary-scheduler-operator:4.12.0 quay.io/openshift/origin-secondary-scheduler-operator:latest
+registry.ci.openshift.org/origin/4.12:security-profiles-operator quay.io/openshift/origin-security-profiles-operator:4.12 quay.io/openshift/origin-security-profiles-operator:4.12.0 quay.io/openshift/origin-security-profiles-operator:latest
```

Comment:
This should probably use `cluster-security-profiles-operator` based on [the guidelines](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md).

If this is part of https://github.com/openshift/enhancements/pull/745 we should get that enhancement wrapped up before doing the image mirroring. Maybe you want to split those changes out of this PR so you can get CI going while we figure out the right image name?

Reply:
OK, I'll admit I haven't read the guidelines properly, so I agree do split the image mirroring and the CI setup.

Is it correct to remove `promotion:` and `releases:` from `ci-operator/config/openshift/security-profiles-operator/openshift-security-profiles-operator-main.yaml` and re-run `make update` ?

Thank you for your help!

Reply:
Or, maybe I was even more confused... SPO is not part of the release payload, but an add-on, like e.g. the Compliance Operator or the Cluster-Logging-Operator. The builds are managed through CPaaS. Do we need to handle image mirroring here at all in that case?

Reply:
If you aren't shipping this in the OCP release, then the name you have now is good.

Reply:
great, marking this as resolved